spec/controllers/albums_controller_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe AlbumsController, type: :controller do
+
+  let(:medium) do
+    Album.create(create_params[:album])
+  end
+
+  it_behaves_like "a medium controller", "album_path", Album, "albums_path"
```

Comment:
You taught ME something: nice use of it_behaves_like in RSpec to help DRY this up.